packages/logger/src/rld-filter.ts

```diff
@@ -0,0 +1,148 @@
+import { Action, Atom, AtomMaybe, Ctx, action, atom } from '@reatom/core'
+import { h } from '@reatom/jsx'
+import styled from 'stylerun'
+import * as model from './rld-filter-model'
+import { t } from './t'
+import { match } from '@reatom/lens'
+import { buttonStyles } from './rld'
+
+export function RldFilterMenu() {
+  return atom((ctx) =>
+    t.div({
+      ...menuStyles({}),
+      children: [
+        RldFilterMenuDraft(),
+        match(ctx.get(model.filters).length, t.div(draftHrStyles({}))),
+        ...ctx.spy(model.filters).map((filter) =>
+          FilterView({
+            persist: atom((ctx) =>
+              ctx.spy(model.filtersLocal).includes(filter),
+            ),
+            setPersist: action((ctx, next) =>
+              model.filterPersistSet(ctx, filter, next),
+            ),
+            action: filter.action,
+            setAction: filter.action,
+            code: filter.code,
+            setCode: filter.code,
+            buttonLabel: 'remove',
+            buttonClicked: action((ctx) => model.filterSplice(ctx, filter)),
+          }),
+        ),
+      ],
+    }),
+  )
+}
+
+function RldFilterMenuDraft() {
+  return FilterView({
+    action: model.draftAction,
+    setAction: model.draftAction,
+    code: model.draftCode,
+    setCode: model.draftCode,
+    persist: model.draftPersist,
+    setPersist: model.draftPersist,
+    buttonLabel: 'create',
+    buttonClicked: model.draftCreate,
+  })
+}
+
+function FilterView({
+  persist,
+  setPersist,
+  action: actionAtom,
+  setAction,
+  code,
+  setCode,
+  buttonLabel,
+  buttonClicked,
+}: {
+  persist: AtomMaybe<boolean>
+  setPersist: (ctx: Ctx, next: boolean) => void
+  action: Atom<model.FilterAction>
+  setAction: (ctx: Ctx, next: model.FilterAction) => void
+  code: AtomMaybe<string>
+  setCode: (ctx: Ctx, next: string) => void
+  buttonLabel: string
+  buttonClicked: (ctx: Ctx) => void
+}) {
+  return h('div', listItemStyles({}), [
+    t.input({
+      ...filterPersistStyles({}),
+      type: 'checkbox',
+      title: 'Persist filter?',
```

Comment:
I was thinking about that... All filters should be persisted by default, but the checkmark should stay and control the "active" behavior. You should be able to deactivate an existing filter easily.